### PR TITLE
2021 83 Release Notes

### PR DESCRIPTION
**Pull request by janettelynch**
Please review and edit.

### CODE REVIEW

**Comment by AnneNielsenJ1** [comment]
@janettelynch I made a few changes, can you re-read to make sure this is still in proper english?

**Comment by AnneNielsenJ1** [comment]
@ninetreats can you review the changes I made and make sure this is still accurate?

File: release-notes/2021-83.md [hunk 1]
```diff
@@ -5,37 +5,37 @@ November 3, 2021
 
 ## New Features and Improvements
 
-- You can now see and review items in the Graph Viewer app that have *problems* 
-  associated with them.
-- The Center for Internet Security (CIS)) Controls Version 8 is released as a compliance 
-  standard within the J1 Compliance app. Additionally, [the controls are mapped](../guides/compliance/compliance-mapping-policies.md) to 
-  the default procedures within the Policies app.
+- Entites in the Graph Viewer that have a *problem* associated with them now have a visual idication and details of the problem
```
Comment (ninetreats) [inline release-notes/2021-83.md:8]: spelling error - "idication"

File: release-notes/2021-83.md [hunk 2]
```diff
@@ -5,37 +5,37 @@ November 3, 2021
...
 
-- You can now see and review items in the Graph Viewer app that have *problems* 
-  associated with them.
-- The Center for Internet Security (CIS)) Controls Version 8 is released as a compliance 
-  standard within the J1 Compliance app. Additionally, [the controls are mapped](../guides/compliance/compliance-mapping-policies.md) to 
-  the default procedures within the Policies app.
+- Entites in the Graph Viewer that have a *problem* associated with them now have a visual idication and details of the problem 
+  in the properties panel. 
+  
+- The Center for Internet Security (CIS)) Controls Version 8 is now available as a standard in the
```
Comment (ninetreats) [inline release-notes/2021-83.md:11]: extra close parens (CIS))

File: release-notes/2021-83.md [hunk 3]
```diff
@@ -5,37 +5,37 @@ November 3, 2021
...
-- You can now see and review items in the Graph Viewer app that have *problems* 
-  associated with them.
-- The Center for Internet Security (CIS)) Controls Version 8 is released as a compliance 
-  standard within the J1 Compliance app. Additionally, [the controls are mapped](../guides/compliance/compliance-mapping-policies.md) to 
-  the default procedures within the Policies app.
+- Entites in the Graph Viewer that have a *problem* associated with them now have a visual idication and details of the problem 
+  in the properties panel. 
+  
+- The Center for Internet Security (CIS)) Controls Version 8 is now available as a standard in the 
+  Compliance with [the mapped controls](../guides/compliance/compliance-mapping-policies.md) to the default procedures
```
Comment (ninetreats) [inline release-notes/2021-83.md:12]: Is there a word missing here? "...in the Compliance with..."

**Review by ninetreats — CHANGES_REQUESTED** [review]
@AnneNielsenJ1 I added a couple comments.  Found some small typos and maybe a missing word.

**Comment by austinkelleher** [comment]
Additional integration release notes @janettelynch @ninetreats :

```md
## Integrations

### Azure

-   Added support for ingesting the following **new** resources:

| Resources                | Entity `_type`             | Entity `_class` |
| ------------------------ | -------------------------- | --------------- |
| Azure Kubernetes Cluster | `azure_kubernetes_cluster` | `Cluster`       |
| Resource Lock            | `azure_resource_lock`      | `Rule`          |
| Key Vault Key            | `azure_keyvault_key`       | `Key`           |
| Key Vault Secret         | `azure_keyvault_secret`    | `Secret`        |

-   Added support for ingesting the following **new** relationships:

| Source Entity `_type`  | Relationship `_class` | Target Entity `_type`      |
| ---------------------- | --------------------- | -------------------------- |
| `azure_resource_group` | **HAS**               | `azure_kubernetes_cluster` |
| `azure_resource_lock`  | **HAS**               | `ANY_SCOPE`                |

-   New properties added to resources:

    | Entity                   | Properties                                                                                                    |
    | ------------------------ | ------------------------------------------------------------------------------------------------------------- |
    | `azure_keyvault_service` | `enableSoftDelete`, `enablePurgeProtection`                                                                   |
    | `azure_storage_account`  | `tableAnalyticsLoggingReadEnabled`, `tableAnalyticsLoggingWriteEnabled`, `tableAnalyticsLoggingDeleteEnabled` |

#### JupiterOne Questions Added

> Are all critical resources protected by resource locks?
> Are all Key Vaults recoverable?
> Is RBAC enabled on all Azure Kubernetes Services Instances?

### GitHub

-   When a JupiterOne GitHub application installation does not have permission to fetch two factor authentication information for `github_user`s, the `github_user` `mfaEnabled` property will now be assigned `undefined` instead of `false`

### Knowbe4

-   Added support for ingesting the following **new** resources:

| Resources                      | Entity `_type`                   | Entity `_class` |
| ------------------------------ | -------------------------------- | --------------- |
| KnowBe4 Phishing Campaign      | `knowbe4_phishing_campaign`      | `Training`      |
| KnowBe4 Phishing Security Test | `knowbe4_phishing_security_test` | `Assessment`    |

-   Added support for ingesting the following **new** relationships:

| Source Entity `_type`       | Relationship `_class` | Target Entity `_type`            |
| --------------------------- | --------------------- | -------------------------------- |
| `knowbe4_account`           | **HAS**               | `knowbe4_phishing_campaign`      |
| `knowbe4_account`           | **HAS**               | `training_campaign`              |
| `knowbe4_phishing_campaign` | **CONTAINS**          | `knowbe4_phishing_security_test` |

### Kubernetes

-   Added support for ingesting the following **new** resources:

| Resources                            | Entity `_type`                   | Entity `_class` |
| ------------------------------------ | -------------------------------- | --------------- |
| Kubernetes Cluster                   | `kube_cluster`                   | `Cluster`       |
| KnowBe4 Phishing Security Test       | `knowbe4_phishing_security_test` | `Assessment`    |
| Kubernetes Container Spec            | `kube_container_spec`            | `Configuration` |
| Kubernetes Pod Security Policy (PSP) | `kube_pod_security_policy`       | `Configuration` |
| Kubernetes Network Policy            | `kube_network_policy`            | `Configuration` |

-   Added support for ingesting the following **new** relationships:

| Source Entity `_type` | Relationship `_class` | Target Entity `_type`      |
| --------------------- | --------------------- | -------------------------- |
| `kube_cluster`        | **CONTAINS**          | `kube_namespace`           |
| `kube_deployment`     | **USES**              | `kube_container_spec`      |
| `kube_cluster`        | **CONTAINS**          | `kube_pod_security_policy` |
| `kube_namespace`      | **CONTAINS**          | `kube_network_policy`      |

#### JupiterOne Questions Added

> Which Kubernetes container specs admit containers to run in privileged mode?
> Which Kubernetes containers are permitted to run as the root user?
> Which Kubernetes Pod Security Policies (PSP) admit privileged containers?
> Which Kubernetes Pod Security Policies (PSP) admit containers to share the host process ID namespace?
> Which Kubernetes Pod Security Policies (PSP) admit containers to share the host IPC namespace?
> Which Kubernetes Pod Security Policies (PSP) admit containers to share the host network namespace?
> Which Kubernetes Pod Security Policies (PSP) admit containers to escalate privileges?
> Which Kubernetes Pod Security Policies (PSP) admit containers with the NET_RAW capability?
> Which Kubernetes Pod Security Policies (PSP) admit containers with the added capabilities?
> Which Kubernetes namespaces do not have network policies defined?

### Qualys

-   Qualys web application scan ingestion is now able to be enabled/disabled based on the integration configuration. A new "Ingest Web App Scans" integration configuration option has been introduced, and the option can be toggled from the JupiterOne Qualys integration configuration page
```

**Comment by austinkelleher** [comment]
Additionally, under the "Qualys" section:

```
- JupiterOne now fetches Qualys findings (`qualys_host_finding`) with status "Fixed"
```

**Review by AnneNielsenJ1 — APPROVED** [review]

**Review by AnneNielsenJ1 — APPROVED** [review]

**Pull request merged**